src/transxchange/naptan.rs

```diff
@@ -356,28 +369,19 @@ mod tests {
 "0100053308","Counterslip","SW-bound",-2.5876736730,51.4557030625,"del""#;
             let mut stop_in_area = HashMap::new();
             stop_in_area.insert(String::from("0100053316"), String::from("stop-area-1"));
-            stop_in_area.insert(String::from("0100053264"), String::from("stop-area-2"));
             stop_in_area.insert(String::from("0100053308"), String::from("stop-area-3"));
-            let stop_points = read_stops(csv_content.as_bytes(), &stop_in_area).unwrap();
+            let (stop_points, stop_areas) =
+                read_stops(csv_content.as_bytes(), &stop_in_area).unwrap();
+
             assert_eq!(stop_points.len(), 2);
             let stop_point = stop_points.get("0100053316").unwrap();
             assert_eq!(stop_point.name, "Broad Walk Shops");
             let stop_point = stop_points.get("0100053264").unwrap();
             assert_eq!(stop_point.name, "Alberton Road");
-        }
 
-        #[test]
-        fn no_stop_area() {
```

Comment:
Note, I merged that test with the above by removing the stop area from the second stop point. And I assert that the resulting `CollectionWithId<StopArea>` contains the newly created `StopArea`.